fhirstore/fhirstore.py

```diff
@@ -290,48 +300,52 @@ def search(self, resource_type, args):
         """
         self.validate_resource_type(resource_type)
 
-        raw_args = SearchArguments(args, resource_type)
+        search_args = SearchArguments()
+        search_args.parse_arguments(args, resource_type)
```

Comment:
no need to repeat "arguments" in the name of the function since it's already the name of the class.
search_args.parse()
same thing for bundle: bundle.initiate()